### PR TITLE
[doc] adding ->createView() in normal symfony controller

### PR DESCRIPTION
**Pull request by bdujon**
Sorry if i did something wrong since it s my first PR there.

If you don't add ->createView() in the render you ll get an error Cannot assign Symfony\Component\Form\Form to property xxxComponent::$formView of type ?Symfony\Component\Form\FormView

### CODE REVIEW

File: src/LiveComponent/doc/index.rst [hunk 1]
```diff
@@ -1482,7 +1482,7 @@ this, create your controller like normal, including the submit logic::
 
             return $this->render('post/edit.html.twig', [
                 'post' => $post,
-                'form' => $form,
+                'form' => $form->createView(),
```
Comment (kbond) [inline src/LiveComponent/doc/index.rst]:
```suggestion
                'form' => $form, // use $form->createView() in Symfony <6.2
```

Not calling `createView()` in [Symfony 6.2+ is valid](https://symfony.com/blog/new-in-symfony-6-2-dx-improvements#simpler-form-rendering), but in <6.2, it is not.

**Review by kbond — CHANGES_REQUESTED** [review]

File: src/LiveComponent/doc/index.rst (hunk 1 above)
Comment (smnandre) [reply src/LiveComponent/doc/index.rst]: Will LiveComponent support 5.4 when it is marked as non experimental ?

Comment (bdujon) [reply src/LiveComponent/doc/index.rst]: > Not calling `createView()` in [Symfony 6.2+ is valid](https://symfony.com/blog/new-in-symfony-6-2-dx-improvements#simpler-form-rendering), but in <6.2, it is not.

Thanks I am indeed using symfony 5.4, i didn't know ->createView() is not needed anymore

**Review by kbond — APPROVED** [review]

**Comment by weaverryan** [comment]
Thanks @bdujon!

**Pull request merged**